src/parsing/Manifest_jsonnet_to_AST_generic.ml

```diff
@@ -12,9 +12,11 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the file
  * LICENSE for more details.
  *)
-module V = Value_jsonnet
+
 module G = AST_generic
 module A = AST_jsonnet
+module E = Eval_jsonnet
+open Value_jsonnet
```

Comment:
again, please avoid using open and instead qualify. 
Here we manipulate 3 different complex data structures, so better qualify.